config-store-api/src/main/java/io/stargate/config/store/api/ConfigWithOverrides.java

```diff
@@ -45,20 +47,35 @@ public ConfigWithOverrides(@Nonnull Map<String, Object> configMap) {
    *
    * <p>3. Underlying config map
    *
+   * <p>When trying to retrieve the system property and environment variable, it will use the full
```

Comment:
System props are prefixed with `stargate.` which I think is important since you can have several `-D` for stargate, C*, logback, ...

Reply:
See my answer here:
https://github.com/stargate/stargate/pull/350#discussion_r514933351